lib/core/src/lib/viewer/components/img-viewer.component.spec.ts

```diff
@@ -354,17 +354,21 @@ describe('Test Img viewer component ', () => {
             component.readOnly = false;
             component.isEditing = true;
 
+            const canvasMock = document.createElement("CANVAS");
             spyOn(component.isSaving, 'emit');
             spyOn(component, 'save').and.callThrough();
+            spyOn(component.cropper, 'getCroppedCanvas').and.returnValue(canvasMock as HTMLCanvasElement);
```

Comment:
I think you don't need this 'as HTMLCanvasElement' here. The reason why canvasMock has incorrect type is because you typed incorrect name in document.createElement. Should be 'canvas' not 'CANVAS'. If you rename it to 'canvas' then you will get proper type, see below screenshot: 
<img width="690" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/8b71d271-b851-406b-8686-54d092224a5f">